cfpb/cfgov-refresh
# Add script to parse through data for external links to search through…

We need icons for links to external sites. This adds the markup for the icons to sheer indexed content, as well as refactors the script to do the processing.

## Changes

- BeautifulSoup 4.4.1
- The script has been refactored to use more robust bs4 methods
- All sheer processors use the external links script now

## Testing

- Run `pip install -r requirements/local.txt`.
- Run `./cfgov/manage.py sheer_index`.
- Run `gulp build`
- Check out [this page](http://localhost:8000/blog/5-things-to-consider-before-you-collect-your-social-security-benefits/) for the icon on the `seven in ten consumers` link but not others.
- Be sure to check out the Wagtail richtextfield. I couldn't easily find a richtextfield in wagtail with an external link but you could go ahead and add that yourself.  

## Review

- @kave 
- @richaagarwal 
- @rosskarchner 
- @jimmynotjim 

## Screenshots
![screen shot 2016-03-22 at 11 09 29 am](https://cloud.githubusercontent.com/assets/1412978/13966334/3c09ebae-f048-11e5-92f6-759dabe451ec.png)
